core/tests/ts-integration/tests/base-token.test.ts

```diff
@@ -22,7 +22,8 @@ describe('base ERC20 contract checks', () => {
         bob = testMaster.newEmptyAccount();
 
         baseTokenDetails = testMaster.environment().baseToken;
-        isETHBasedChain = process.env.CONTRACTS_BASE_TOKEN_ADDR! == zksync.utils.ETH_ADDRESS_IN_CONTRACTS;
+        const baseToken = await alice.provider.getBaseTokenContractAddress();
+        isETHBasedChain = baseToken == zksync.utils.ETH_ADDRESS_IN_CONTRACTS;
```

Comment:
It might make sense to use isAddressEq here as well, but not essential as ETH_ADDRESS_IN_contracts only has numbers